mmdet/models/detectors/glip.py

```diff
@@ -128,7 +168,23 @@ def create_positive_map_label_to_token(positive_map: Tensor,
 
 @MODELS.register_module()
 class GLIP(SingleStageDetector):
-    """Implementation of `GLIP <https://arxiv.org/abs/2112.03857>`_"""
+    """Implementation of `GLIP <https://arxiv.org/abs/2112.03857>`_
+    Args:
+        backbone (:obj:`ConfigDict` or dict): The backbone config.
+        neck (:obj:`ConfigDict` or dict): The neck config.
+        bbox_head (:obj:`ConfigDict` or dict): The bbox head config.
+        language_model (:obj:`ConfigDict` or dict): The language model config.
+        train_cfg (:obj:`ConfigDict` or dict, optional): The training config
+            of FOVEA. Defaults to None.
```

Comment:
FOVEA?

Reply:
Sorry for the typo haha, I'll fix it right now.